app/bundles/EmailBundle/Translations/en_US/messages.ini

```diff
@@ -179,7 +179,7 @@ mautic.email.config.resubscribe_message.tooltip="Set the default text for the me
 mautic.email.config.resubscribe_message="Resubscribed confirmation message"
 mautic.email.config.unsubscribe_message.tooltip="Set the default text for the message displayed to the contact when they unsubscribe using the link in an email. Be sure to use the <code>|URL|</code> placeholder where the resubscribe url is to be injected. The use of <code>|EMAIL|</code> is optional."
 mautic.email.config.unsubscribe_message="Unsubscribed confirmation message"
-mautic.email.config.unsubscribe_text.tooltip="Set the default text for the {unsubscribe_token} used in emails. Be sure to use the <code>|URL|</code> placeholder where the unsubscribe url is to be injected."
+mautic.email.config.unsubscribe_text.tooltip="Set the default text for the {unsubscribe_text} used in emails. Be sure to use the <code>|URL|</code> placeholder where the unsubscribe url is to be injected."
```

Comment:
Then i'd say:
> Set the default text for the {unsubscribe_text} token used in emails.

Reply:
Initially I also thought so, but then I just adjusted with webview_text tooltip. If so, it would be nice if we do the same thing on webview_text and signature.